app/routes/events/components/EventDetail/index.tsx

```diff
@@ -342,20 +343,46 @@ export default class EventDetail extends Component<Props, State> {
           </span>
         ),
       },
-      event?.createdBy
-        ? event.createdBy && {
-            key: 'Forfatter',
-            value: (
-              <Link to={`/users/${event.createdBy.username}`}>
-                {event.createdBy.fullName}
-              </Link>
-            ),
-          }
-        : {
-            key: 'Forfatter',
-            value: 'Anonym',
-          },
-    ];
+
+      // Responsible Users or Created By or Anonymous
+      ...(event.responsibleUsers && event.responsibleUsers.length > 0
+        ? [
+            {
+              key:
+                event.responsibleUsers.length > 1
+                  ? 'Kontaktpersoner'
+                  : 'Kontaktperson',
+              value: (
+                <ul>
+                  {event.responsibleUsers.map((user) => (
+                    <li key={user.id}>
+                      <Link to={`/users/${user.username}`} key={user.username}>
+                        {user.fullName}
+                      </Link>
+                    </li>
+                  ))}
+                </ul>
+              ),
+            },
+          ]
+        : event.createdBy
+        ? [
+            {
+              key: 'Forfatter',
+              value: (
+                <Link to={`/users/${event.createdBy.username}`}>
+                  {event.createdBy.fullName}
+                </Link>
+              ),
+            },
+          ]
+        : [
+            {
+              key: 'Forfatter',
+              value: 'Anonym',
+            },
+          ]),
```

Comment:
So our thought was to basically just remove this side of the ternary and keep the previous lines

Reply:
Ok great I'll implement the changes and get it merged 👍